app/controllers/events/search_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Events::SearchController < ApplicationController
+  def index
+    @events = Event.near(get_location, get_radius).map{ |event| EventPresenter.new(event)}
```

Comment:
The near method is something we get from geocoding the events and trails. They are geocoded now when they are created and it means we can pass in a location coordinate and a radius and it will return a collection of events within that raidus(as the crow flies, not street distance).